firmware/main/conductor/conductor.ino

```diff
@@ -64,10 +307,56 @@ void setup() {
   staff.setDebugVerbose(DEBUG_VERBOSE);
 
   playbackTimer.begin(playColumn, PLAYBACK_PERIOD_DEFAULT);
+
+  Serial.begin(9600);
+  AudioMemory(40);
+  sgtl5000_1.enable();
+  sgtl5000_1.volume(.8);
+  SPI.setMOSI(SDCARD_MOSI_PIN);
+  SPI.setSCK(SDCARD_SCK_PIN);
+  if (!(SD.begin(SDCARD_CS_PIN))) {
+    while (1) {
+      Serial.println("Unable to access the SD card");
+      delay(500);
+    }
+  }
+  pinMode(13, OUTPUT); // LED on pin 13
+  
+  mixer1.gain(0, .1);
+  mixer1.gain(1, .1);
+  mixer1.gain(2, .1);
+  mixer1.gain(3, .1);
+  
+  mixer2.gain(0, .1);
+  mixer2.gain(1, .1);
+  mixer2.gain(2, .1);
+  mixer2.gain(3, .1);
+  
+  mixer3.gain(0, .1);
+  mixer3.gain(1, .1);
+  mixer3.gain(2, .1);
+
+  mixer4.gain(0, .1);
+  mixer4.gain(1, .1);
+  mixer4.gain(2, .1);
+ 
+  mixer5.gain(0, .1);
+  mixer5.gain(1, .1);
+  
+  mixer6.gain(0, .1);
+  mixer6.gain(1, .1);
+  
+  
+
+  
 }
 
 void loop() {
-
+  //if (!(SD.begin(SDCARD_CS_PIN))) {
+  //  while (1) {
+  //    Serial.println("Unable to access the SD card");
+      delay(500);
+  //  }
+  //}
```

Comment:
This block and delay will stall the conductor receiving messages from the network hub. I think we can remove this entire block, comments and all.